tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/utils/ProjectHasNature.java

```diff
@@ -45,6 +46,13 @@ public boolean test() {
 		pd.open();
 		new WaitUntil(new ShellIsAvailable("Properties for " + projectName), TimePeriod.DEFAULT);
 		new DefaultTreeItem("Project Facets").select();
+		if (!getJavaVersion().equals("1.8")) {
```

Comment:
You should add comment with reason for this hack and also why you not doing anything when exception is thrown.

Reply:
Comment added